scripts/options.js

```diff
@@ -146,14 +146,14 @@ function save() {
     if (document.querySelector('.branch-button input[type=checkbox]').checked) {
       settings.branch = getButton(document.querySelector('.branch-button'));
     }
-  document.getElementById('result').innerHTML = '## [UI Previewer Button](https://rightscale.github.io/ui-previewer-button/) config\n\n```json\n' + JSON.stringify(settings, null, 2) + '\n```';
+  document.getElementById('result').innerHTML = '### [UI Previewer Button](https://rightscale.github.io/ui-previewer-button/) config\n\n```json\n' + escapeHtml(JSON.stringify(settings, null, 2)) + '\n```';
```

Comment:
The backslashes are taken care of by the JSON encoder, it's the angle brackets that need taking care of, otherwise the page tries to render an invalid svg tag in the middle of the JSON.